Sources/SRGDataProviderNetwork/SRGDataProvider+ContentServices.m

```diff
@@ -51,6 +51,28 @@ - (SRGRequest *)contentPageForVendor:(SRGVendor)vendor
     return [self fetchObjectWithURLRequest:URLRequest modelClass:SRGContentPage.class completionBlock:completionBlock];
 }
 
+- (SRGRequest *)contentPageForVendor:(SRGVendor)vendor
+                             product:(SRGProduct)product
+                         showWithURN:(NSString *)showURN
+                           published:(BOOL)published
+                              atDate:(NSDate *)date
+                 withCompletionBlock:(SRGContentPageCompletionBlock)completionBlock
+{
+    NSURLRequest *URLRequest = [self requestContentPageForVendor:vendor productName:product showWithURN:showURN published:published atDate:date];
+    return [self fetchObjectWithURLRequest:URLRequest modelClass:SRGContentPage.class completionBlock:completionBlock];
+}
+
+- (SRGRequest *)contentPageForVendor:(SRGVendor)vendor
+                         productName:(NSString *)productName
+                         showWithURN:(NSString *)showURN
+                           published:(BOOL)published
+                              atDate:(NSDate *)date
+                 withCompletionBlock:(SRGContentPageCompletionBlock)completionBlock
+{
+    NSURLRequest *URLRequest = [self requestContentPageForVendor:vendor productName:productName showWithURN:showURN published:published atDate:date];
+    return [self fetchObjectWithURLRequest:URLRequest modelClass:SRGContentPage.class completionBlock:completionBlock];
+}
+
```

Comment:
Is it really useful to have these two methods? Since `SRGProduct` is an alias for `NSString`, we could probably just keep the version with the product name. 🤷‍♂️

Reply:
Good question.
I didn't challenge this question and follow what did on other entry, so that it's consistence.

ie: in the whole file Sources/SRGDataProviderNetwork/SRGDataProvider+ContentServices.m

@defagos, As I remember, first iteration supports only official `SRGProduct` with a define list of product string. But then, PAC introduced more product.

It's an opened question to remove `SRGProduct` or follow new product string from the official PAC project.
https://github.com/mmz-srf/play-pac/blob/88fe69819fb342f8c3b18d38da31d26dc79795b0/src/main/kotlin/ch/srg/pac/domain/shared/Product.kt#L6

```
/**
 * Supported Products, default is PLAY_VIDEO
 */
enum class Product(val value: String) {
    PLAY_VIDEO("play_video"),
    PLAY_AUDIO("play_audio"),
    NEWS_VIDEO("news_video"),
    NEWS_AUDIO("news_audio");

   […]
}
```

Reply:
Issue created to do it on the whole library.
https://github.com/SRGSSR/srgdataprovider-apple/issues/53